crypto/params.c

```diff
@@ -219,9 +221,13 @@ int OSSL_PARAM_set_int32(OSSL_PARAM *p, int32_t val)
             p->return_size = sizeof(int64_t);
             *(int64_t *)p->data = (int64_t)val;
             return 1;
+        case 0:
```

Comment:
???

Wouldn't the switch fall through without this?

I.e. remove the case.

Reply:
That was from earlier development, before I had properly wrapped my brain around this code.

Gone